cmd/ssl-vision-client/main.go

```diff
@@ -99,7 +99,7 @@ func setupUi() {
 
 	http.Handle("/", withResponseHeaders(http.FileServer(box)))
 	if box.Has("index.html") {
-		log.Printf("UI is available at http://%v", *address)
+		log.Printf("UI is available at http://localhost%v", *address)
```

Comment:
`*address` can also be `localhost:8080`. The default is `:8080`, which means you can also access is from an external IP (e.g. in docker).

A more correct way would be to parse `*address` by splitting it at `:` and default to `localhost`, if the host part is empty.

Reply:
hmm ok. I think I get it.